src/main/java/org/broadinstitute/hellbender/tools/walkers/sv/SVAnnotateEngine.java

```diff
@@ -620,7 +717,49 @@ protected static List<SVSegment> getSVSegments(final VariantContext variant,
         return intervals;
     }
 
+    /**
+     * Returns a new subsetted list of SVSegments to use for promoter & noncoding annotations for complex SVs.
+     * Removes DUP segments which are never tandem in CPX events
+     * @param svSegments - List of SVSegments used for gene overlap annotations
+     * @return - Subsetted list of SVSegments to use for promoter & noncoding annotations for CPX SVs
+     */
+    @VisibleForTesting
+    protected static List<SVSegment> getSegmentsForNonCodingAnnotations(final List<SVSegment> svSegments) {
```

Comment:
I updated these functions to run on all SV types instead of just complex, to go with the other change you suggested below to create new lists of segments for each type of annotation